e2e/config.yaml.sample

```diff
@@ -1,11 +1,35 @@
 ---
+# Configuration file for RamenDR E2E testing on Regional DR.
+
+# Name of the channel used for the Channel CR (utilized by Subscription CRs).
```

Comment:
Also by ApplicationSet, so we can just mention that OCM is using this to pull the application from git.

Reply:
ack, changed